plugins/rapid7_insightidr/plugin.spec.yaml

```
@@ -650,9 +747,11 @@ types:
     attachments:
       title: Attachments
       description: List of attachments associated with this comment
-      type: "[]attachment"
+      type: '[]attachment'
       required: false
-      example: [{"rrn": "rrn:collaboration:us:44d88612-fea8-a8f3-6de8-2e1278abb02f:attachment:1234567890","creator": {"type": "USER","name": "Example User"},"created_time": "2022-08-19T13:00:58.645Z","file_name": "test.txt","mime_type": "text/plain","size": 4,"scan_status": "CLEAN"}]
+      example: [{rrn: rrn:collaboration:us:44d88612-fea8-a8f3-6de8-2e1278abb02f:attachment:1234567890,
```

Ensure these are in quotes and also ensure it isn't sensitive
```suggestion
      example: [{"rrn": "rrn:collaboration:us:44d88612-fea8-a8f3-6de8-2e1278abb02f:attachment:1234567890"}]
```